lib/eu/hydrologis/smash/mainview.dart

```diff
@@ -174,9 +182,12 @@ class MainViewWidgetState extends State<MainViewWidget> with WidgetsBindingObser
 
   WillPopScope consumeBuild(SmashMapBuilder mapBuilder) {
     var layers = <LayerOptions>[];
-    _iconSize = GpPreferences().getDoubleSync(SmashPreferencesKeys.KEY_MAPTOOLS_ICON_SIZE, SmashUI.MEDIUM_ICON_SIZE);
-    var projectState = Provider.of<ProjectState>(mapBuilder.context, listen: false);
-    var mapState = Provider.of<SmashMapState>(mapBuilder.context, listen: false);
+    _iconSize = GpPreferences().getDoubleSync(
```

Comment:
we will need to understand what is different in our formatting, else we will have to check 1000s of these while reviewing.